plugins/buildSrcLibs/src/main/kotlin/de/fayard/buildSrcLibs/BuildSrcLibsPlugin.kt

```diff
@@ -0,0 +1,20 @@
+package de.fayard.buildSrcLibs
+
+import org.gradle.api.Plugin
+import org.gradle.api.initialization.Settings
+import org.gradle.kotlin.dsl.register
+
+class BuildSrcLibsPlugin : Plugin<Settings> {
```

Comment:
@LouisCAD is it intentional that you make it a Plugin of Settings instead of a normal plugin?
I don't see why that would be needed

Reply:
Yes, it's intentional, but I might make it a `Plugin<Any>` depending on the outcome of my experiments.

Anyway, it's going to apply the refreshVersions plugin automatically.

Reply:
I have set-up JetPack for development version, but because the plugin is a `Plugin<Settings>`, I don't know how to configure it correctly =>https://github.com/jmfayard/refreshVersions/issues/290

Reply:
@jmfayard The generic type parameter is not relevant to the maven publication, it shouldn't be different from another plugin.

If you are asking about getting the maven coordinates from the plugin id, there's an image and details about this on that page: https://docs.gradle.org/current/userguide/plugins.html